crates/anvil/src/eth/miner.rs

```diff
@@ -25,12 +25,20 @@ pub struct Miner {
     ///
     /// This will register the task so we can manually wake it up if the mining mode was changed
     inner: Arc<MinerInner>,
+    // TODO(serge): ...
+    //replay_transactions: Option<Vec<Arc<PoolTransaction>>>,
+    replay_transactions: Option<Vec<PoolTransaction>>,
```

Comment:
oh this is actually a better solution than I suggested.

let's find a more appropriate name here that makes it clear that these will be forced returned next

Reply:
updated